src/ign.cc

```diff
@@ -50,6 +54,59 @@ extern "C" IGNITION_GAZEBO_VISIBLE const char *worldInstallDir()
   return IGN_GAZEBO_WORLD_INSTALL_DIR;
 }
 
+//////////////////////////////////////////////////
+extern "C" IGNITION_GAZEBO_VISIBLE const char *findFuelResource(
+    char *_pathToResource)
+{
+  std::string path;
+  std::string worldPath;
+  ignition::fuel_tools::ServerConfig server;
+  server.SetUrl(ignition::common::URI("https://fuel.ignitionrobotics.org"));
+  ignition::fuel_tools::ClientConfig config;
+  config.AddServer(server);
+  ignition::fuel_tools::FuelClient fuelClient(config);
+
+  // Attempt to find cached copy, and then attempt download
+  if (fuelClient.CachedWorld(ignition::common::URI(_pathToResource), path))
+  {
+    worldPath = path;
+  }
+  else if (ignition::fuel_tools::Result result =
+    fuelClient.DownloadWorld(ignition::common::URI(_pathToResource), path);
+    result)
+  {
+    worldPath = path;
+  }
+  else
+  {
+    ignwarn << "Fuel world download failed because " << result.ReadableResult()
+        << std::endl;
+    return "";
+  }
+
+  // Find the first sdf file in the world path
+  if (ignition::common::exists(worldPath))
+  {
+    for (ignition::common::DirIter file(worldPath);
```

Comment:
Can you add a note here that we're getting the first `.sdf` file for now?

Reply:
63eb1b0b94e490d457fbe808695fa5062239479e